com.sublimetext.three.json

```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "com.sublimetext.three",
     "runtime": "org.gnome.Sdk",
-    "runtime-version": "3.24",
+    "runtime-version": "3.26",
     "sdk": "org.gnome.Sdk",
     "branch": "1.0",
```

Comment:
More of an FYI but when this moves to flathub this'll have to change to 'stable'